MMCore/MMCore.cpp

```diff
@@ -2658,6 +2691,203 @@ bool CMMCore::getShutterOpen() throw (CMMError)
    return getShutterOpen(shutterLabel.c_str());
 }
 
+/**
+ * A centralized function that adds all the metadata for camera devices.
+ * 
+ * This was previously spread among the circular buffer, corecallback.h, and 
+ * the SWIG wrapper.
+ * 
+ * Get the metadata tags attached to device caller, and merge them with metadata
+ * in pMd (if not null). Returns a metadata object.
+ * 
+ * The caller should have locked the camera device, or be calling from a thread
+ * in the camera (e.g. CoreCallback::InsertImage)
+ */
+void CMMCore::addCameraMetadata(std::shared_ptr<CameraInstance> pCam, Metadata& md, unsigned width, unsigned height,
```

Comment:
```
      mmCoreAndDevices/MMCore/MMCore.cpp:2706:15: error: exception specification in declaration does not match previous declaration
       2706 | void CMMCore::addCameraMetadata(std::shared_ptr<CameraInstance> pCam, Metadata& md, unsigned width, unsigned height,
            |               ^

```

Reply:
not sure about this. still seeing it with latest code?